tox.ini

```diff
@@ -1,21 +1,26 @@
 [tox]
 envlist =
-    py{26}-django{15,16},
-    py{27,33,34}-django{15,16,17,18},
+    py{26,27,32,33}-django{15,16},
+    py{27,32,33,34}-django{17,18},
 
 [testenv]
+changedir = demo_project
 commands =
-    coverage run setup.py test
-    coveralls
-basepython =
-    py26: python2.6
-    py27: python2.7
-    py33: python3.3
-    py34: python3.4
+    python manage.py syncdb --noinput
+    python manage.py validate
+    # TODO: for Django>=1.7 execute instead: ===
```

Comment:
Yep, I've actually already updated to native migrations in a branch. Should go into master shortly.